src/api/product-pagination.ts

```diff
@@ -0,0 +1,25 @@
+import { IProductPagination } from './types';
+
+export const DEFAULT_PAGE_SIZE = 50;
+
+export const OFFSET_SEARCH_PARAM = 'offset';
+export const LIMIT_SEARCH_PARAM = 'limit';
+
+export function productPaginationFromSearchParams(searchParams: URLSearchParams): IProductPagination {
+    const offset = searchParams.get(OFFSET_SEARCH_PARAM);
+    const limit = searchParams.get(LIMIT_SEARCH_PARAM);
```

Comment:
rename search params to `currentPage` and `productsPerPage` or something similar as it makes more sense from the user's perspective

Reply:
but these search params are internal, user is not seeing them. they are passed to API endpoint only.
also, I'd recommend to stay with offset/limit approach rather then pageNumber/pageSize, because it is more flexible. also, it will require less calculations, cause SDK accept this parameters, not page number/size